secboot/argon2_out_of_process_dummy.go

```diff
@@ -0,0 +1,23 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build nosecboot
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package secboot
+
+func MaybeRunArgon2OutOfProcessRequestHandler() {}
```

Comment:
```suggestion
func MaybeRunArgon2OutOfProcessRequestHandler() {
	if !isOutOfProcessArgon2KDFMode() {
		panic("internal error: unexpected call to execute as argon2 runner in non-secboot binary")
	}
}
```

Reply:
I think it should panic when `isOutOfProcessArgon2KDFMode` is true